src/crimson/os/seastore/onode_manager/staged-fltree/tree.h

```diff
@@ -221,9 +221,9 @@ class Btree {
   eagain_ifuture<Cursor> lower_bound(Transaction& t, const ghobject_t& obj) {
     return seastar::do_with(
       full_key_t<KeyT::HOBJ>(obj),
-      [this, &t](auto& key) -> eagain_ifuture<Cursor> {
-        return get_root(t).si_then([this, &t, &key](auto root) {
-          return root->lower_bound(get_context(t), key);
+      [this, &t, &obj](auto& key) -> eagain_ifuture<Cursor> {
+        return get_root(t).si_then([this, &t, &key, &obj](auto root) {
```

Comment:
Ditto, `obj` is not supposed to have extented lifecycle beyond async operations.